python/cuspatial/cuspatial/core/binpreds/binpred_dispatch.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+
+from cuspatial.core.binpreds.feature_contains import (
+    DispatchDict as CONTAINS_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_covers import (
+    DispatchDict as COVERS_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_crosses import (
+    DispatchDict as CROSSES_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_equals import (
+    DispatchDict as EQUALS_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_intersects import (
+    DispatchDict as INTERSECTS_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_overlaps import (
+    DispatchDict as OVERLAPS_DISPATCH_DICT,
+)
+from cuspatial.core.binpreds.feature_within import (
+    DispatchDict as WITHIN_DISPATCH_DICT,
+)
+
+CONTAINS_DISPATCH = CONTAINS_DISPATCH_DICT
```

Comment:
Why not just do

```python
from cuspatial.core.binpreds.feature_contains import (
    DispatchDict as CONTAINS_DISPATCH,
)
```
?

Reply:
Good idea. I was "having variables" in this collector module, why not just go directly to the imports.